src/features/Search/Search.tsx

```diff
@@ -22,8 +20,8 @@ import theme from '../../shared/Styles/theme';
 import { getNestedAttr, getValueFromQuery, isSponsor } from '../../util';
 
 import withContext from '../../shared/HOC/withContext';
-import { FilterComponent } from './Filters';
-import { ResultsTable } from './ResultsTable';
+import { FilterComponent } from '../../features/Search/Filters';
+import { ResultsTable } from '../../features/Search/ResultsTable';
```

Comment:
why the change here if they're in the same folder?

Reply:
yup ill fix it